vulnerable_people_form/form_pages/support_address.py

```diff
@@ -10,8 +10,9 @@
 
 @form.route("/support-address", methods=["POST"])
 def post_support_address():
-    original_address = {k: v for k, v in form_answers()["support_address"].items() if k != "uprn"}
+    original_address = {k: v for k, v in form_answers().get("support_address", {}).items() if k != "uprn"}
```

Comment:
it might be best to declare get the support address first and check for it's existence before accessing items()

Reply:
it defaults to an empty object if there is no support address so items will always work